lib/timer.py

```diff
@@ -117,5 +112,17 @@ def run_jobs(cfg: UAConfig, current_time: datetime):
 if __name__ == "__main__":
     cfg = UAConfig()
     current_time = datetime.utcnow()
-    setup_logging(logging.INFO, logging.DEBUG)
+
+    setup_logging(
+        logging.CRITICAL,
+        logging.DEBUG,
+        log_file="/var/log/ubuntu-advantage-timer.log",
+        logger=LOG,
+    )
+    LOG.propagate = False
+
+    # Root logger should still send errors to the log file,
+    # but nothing to console here
+    setup_logging(logging.CRITICAL, logging.ERROR)
```

Comment:
Do we need to setup the root logger as well ?

Reply:
Yes. As we are running jobs that might have their own logging, we want to capture it when those are executed - any error would go to the general log file.
An alternative here would be configuring only one logger, and log everything (including eventual errors in executed code  inside of the jobs) to the timer file.
What do you think? Should we have it all together?